packages/cli-helpers/src/lib/project.ts

```diff
@@ -33,21 +36,113 @@ export const getInstalledRedwoodVersion = () => {
   }
 }
 
-export const addEnvVarTask = (name: string, value: string, comment: string) => {
+/**
+ * Updates the project's redwood.toml file to include the specified packages plugin
+ *
+ * Uses toml parsing to determine if the plugin is already included in the file and
+ * only adds it if it is not.
+ *
+ * Writes the updated config to the file system by appending strings, not stringify-ing the toml.
+ */
+export const updateTomlConfig = (packageName: string) => {
+  const redwoodTomlPath = getConfigPath()
+
+  const configLines = []
+
+  const configContent = fs.readFileSync(redwoodTomlPath, 'utf-8')
+  const config = toml.parse(configContent)
+
+  if (!config || !config.experimental || !config.experimental.cli) {
+    configLines.push('[experimental.cli]')
+  }
+
+  if (!config?.experimental?.cli?.autoInstall) {
+    configLines.push('  autoInstall = true')
+  }
+
+  if (!config?.experimental?.cli?.plugins) {
+    // If plugins array is missing, create it.
+    configLines.push('  [[experimental.cli.plugins]]')
```

Comment:
I think for TOML we need this array heading for every entry. This would mean even though this `[[experimental.cli.plugins]]` exists in the config file we would need to add another if we have to add another package.

That probably isn't clear so let me add an example. This could be the starting TOML file:
```toml
[experimental.cli]
  autoInstall = false
  [[experimental.cli.plugins]]
    package = "something-the-user-added"
```
If we wanted to add a new CLI package then I think we would need:
```toml
[experimental.cli]
  autoInstall = false
  [[experimental.cli.plugins]]
    package = "something-the-user-added"
  [[experimental.cli.plugins]]
    package = "new-package-added"
```

This is an array of tables in the TOML spec: https://toml.io/en/v1.0.0#array-of-tables